apps/foundation-demo/src/app/checkout/components/print-order-summary/print-order-summary.component.html

```diff
@@ -1 +1 @@
-<button type="button" class="button large expanded" (click)="printOrderSummary()">Print Order Summary</button>
+<button type="button" daff-button color="black" (click)="printOrderSummary()">Print Order Summary</button>
```

Comment:
The associated scss file can be deleted.

Reply:
Resolved in 5c2a675